### PR TITLE
Bootstrap improvements

### PR DESCRIPTION
**Pull request by rjw57**
This PR just consists of fixes/improvements to the bootstrap application. See each commit for details.

Note to reviewer: if you approve this PR, please open an issue on https://github.com/uisautomation/django-boilerplate to port these changes over.

Some changes (e.g. django-ucamlookup and DRF) may or may not want putting into the boilerplate or may want putting behind some configuration.

### CODE REVIEW

File: project/settings/base.py [hunk 1]
```diff
@@ -27,6 +27,11 @@
 
     'automationcommon',
     'ucamwebauth',
+    'crispy_forms',
```
Comment (abrahammartin) [inline project/settings/base.py]: Based on https://django-filter.readthedocs.io/en/master/guide/rest_framework.html?highlight=%22django-crispy-forms%22 this is a bit flacky "Note that this isn’t actively supported, although pull requests for bug fixes are welcome." I'd include a note to bear that in mind as it seems that the project is a bit dead: https://github.com/django-crispy-forms/django-crispy-forms/pulls

**Review by abrahammartin — APPROVED** [review]
just one tiny comment

File: project/settings/base.py (hunk 1 above)
Comment (rjw57) [reply project/settings/base.py]: OK, we can probably remove it.

**Comment by abrahammartin** [comment]
Created corresponding task in boilerplate

**Pull request merged**